infra/gcp/monitoring/k8s.io/10-thockin-k8s-io.tf

```diff
@@ -0,0 +1,75 @@
+locals {
+  project_id       = "kubernetes-public"
+  monitored_domain = "thockin-test1.k8s.io"
+}
+
+// Needed to import the notification channel
+provider "google" {
+  project = local.project_id
+}
+
+// Manual step: Create a StackDriver alert channel pointing to a channel in Slack
+// It will select the channel here by its display name
+data "google_monitoring_notification_channel" "alertchannel" {
+  display_name = "Kubernetes.io Cert Alert"
```

Comment:
We can use the existing channel : `#k8s-infra-alerts`. Can you mention the type ?